src/Arcus.Messaging.Abstractions/MessageHandling/MessageCorrelationEnricherOptions.cs

```diff
@@ -1,16 +1,60 @@
 ﻿using System;
 using Arcus.Messaging.Abstractions.Telemetry;
-using Arcus.Observability.Telemetry.Serilog.Enrichers.Configuration;
 
 namespace Arcus.Messaging.Abstractions.MessageHandling
 {
     /// <summary>
     /// Represents the consumer configurable options model to change the behavior of the Serilog <see cref="MessageCorrelationInfoEnricher"/>.
     /// </summary>
     [Obsolete("Will be removed in v3.0 as W3C will be the only supported correlation format")]
```

Comment:
Next release will be 3.0 isn't it, as we've already created a 3.0-preview package, so can't we just remove this class ?